### PR TITLE
BNGP-4419: Check and submit : multiple Landownership uploads create a new row instead of one and a list of upload

### PR DESCRIPTION
**Pull request by usman-j-shaikh**

### CODE REVIEW

File: packages/webapp/src/routes/land/check-and-submit.js [hunk 1]
```diff
@@ -74,13 +74,14 @@ const getContext = request => {
     localPlanningAuthorities: getLocalPlanningAuthorities(request.yar.get(constants.redisKeys.PLANNING_AUTHORTITY_LIST)),
     ...geospatialOrLandBoundaryContext(request),
     ...applicationInformationContext(request.yar),
-    landownershipFilesRows: getLandOwnershipRows(request.yar.get(constants.redisKeys.LAND_OWNERSHIP_PROOFS)),
+    // landownershipFileNames: getFileNameByType(applicationDetails.files, 'land-ownership'),
```
Comment (paulsimonandrews) [inline packages/webapp/src/routes/land/check-and-submit.js:77]: Remove comment

**Review by paulsimonandrews — CHANGES_REQUESTED** [review]
Tiny change to remove commented out code.

**Review by paulsimonandrews — APPROVED** [review]
LGTM

**Comment by shaundefra** [comment]
tested by Tajinder

**Pull request merged**